public_html/wp-content/mu-plugins/blocks/assets/src/shared/featured-image/index.js

```diff
@@ -0,0 +1,185 @@
+				} else {
+					return 1;
+				}
+			}
+		);
+
+		this.setState(
+			{ sizes }
+		);
+	}
+
+
+	/**
+	 * Returns URL of appropriate image. We find appropriate by looking for an image with similar aspect ratio, and similar or greater area.
+	 * If there is no image with similar aspect ratio or area, we will return strictly bigger image then currently requested.
+	 * If there is no bigger image then currently request one, we will return the max size image URL.
+	 * Note: Core Image block allows user to select image size by name, but we cannot really do that because we will be controlling multiple images at once, and one or more images may not have all the available sizes. So an automated solution is preferred here.
+	 *
+	 * @param height
+	 * @param width
+	 */
+	getSizedUrl( height, width ) {
+		const { sizes } = this.state;
+		if ( ! sizes ) {
+			return "";
+		}
+
+		// Lets loop through all possible images, see if we have an image that matches requested aspect ration. if we have, we will return it. Also keep track of image which has the most closest aspect ration.
+		let smallestImage;
+		let selectedImage;
+		const requiredAspectRatio = width / height;
+
+		for ( const size of sizes ) {
+
+			// Reject all the images with less height or width then 20px.
+			if (size.height < height - 20 || size.width < width - 20) {
+				continue;
+			}
+
+			// Keep track of smallest image that is similar or bigger than our requirements. In case we won't find any image with similar aspect ratio, we will use this.
+			// Since images are sorted by size, this will be the first image that is not rejected by the size filter above.
+			if ( ! smallestImage ) {
+				smallestImage = size;
+			}
+
+			if (size.aspectRatio > requiredAspectRatio - 0.25 &&
+				size.aspectRatio < requiredAspectRatio + 0.25) {
+				// This image has similar aspect ratio to what we need. Also, since possibleImage array is sorted from lowest size, this is also the lowest size image that we can use. Lets go ahead and use this image.
+				selectedImage = size;
+				break;
+			}
+		}
+
+		if ( selectedImage ) {
+			return selectedImage.source_url;
+		} else if ( smallestImage ) {
+			return smallestImage.source_url;
+		} else {
+			// All images are small, lets just return the largest image url.
+			return sizes[ sizes.length - 1 ].source_url;
+		}
+
+	}
+
+	/**
+	 * Renders FeaturedImage component.
+	 *
+	 * @returns {*}
+	 */
+	render() {
+		const { className, alt, onChange } = this.props;
+
+		const imageSize = this.props.size || { height: 150, width: 150 };
+
+		const { height, width } = imageSize;
+
+		const imageURL = this.getSizedUrl( height, width );
+		if ( imageURL !== this.currentImageURL ) {
+			this.currentImageURL = imageURL;
+			if ( typeof( onChange ) === 'function' ) {
+				onChange( imageURL );
+			}
+		}
+		return(
+			<img
+				className={ classnames( 'featured-image', className ) }
+				src = { imageURL }
+				alt = { alt }
+				style = { { height: height + "px"} }
```

Comment:
IIRC height attribute directly was not working because of an overzealous CSS rule.

Reply:
Ugh, I hate it when that happens. I do think, however, the HTML tag and attributes should be the same on the front and back end, and that we should use `get_the_post_thumbnail` for the PHP front end version, because it handles all the stuff around sizing and srcset.

There are some other ways around overly-broad CSS rules. Here's what I did for one case in Speakers: https://github.com/WordPress/wordcamp.org/blob/master/public_html/wp-content/mu-plugins/blocks/assets/src/speakers/block-content.scss#L8-L14

Reply:
The corresponding checkbox for this is marked off in #44, but this is still marked as unresolved. If it's resolved, can you leave a comment w/ the commit that resolved it, and then mark as resolved?